ipld/amt/fuzz/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+
+[package]
+name = "ipld_amt-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
```

Comment:
what does this mean btw

Reply:
It's just what cargo fuzz generates when creating a target